### PR TITLE
Utilise Auth0->Login to ensure state validation

### PR DESCRIPTION
**Pull request by cocojoe**
Update auth0-PHP dependency

### CODE REVIEW

**Comment by cocojoe** [comment]
There is no auth0-PHP 5.1 yet, release that first and can then test this for a quick release.

**Comment by cocojoe** [comment]
Don't merge.

File: src/Auth0/Login/Auth0Service.php [hunk 1]
```diff
@@ -51,10 +51,10 @@ public function logout()
     /**
      * Redirects the user to the hosted login page
      */
-    public function login($connection = null, $state = null, $aditional_params = ['scope' => 'openid profile email'], $response_type = 'code')
+    public function login($connection = null, $state = null, $additional_params = ['scope' => 'openid profile email'], $response_type = 'code')
```
Comment (lbalmaceda) [inline src/Auth0/Login/Auth0Service.php]: This looks at first like a *BREAKING CHANGE*. Is it just the parameter name, used internally in this method logic or can people actually call this method passing a dictionary with the `${param_name}` as keys?

Comment (joshcanhelp) [reply src/Auth0/Login/Auth0Service.php]: @lbalmaceda - No fancy stuff here, just the internal signature.

**Pull request merged**